.github/workflows/common-tests.jl

```diff
@@ -0,0 +1,11 @@
+if !occursin("hostedtoolcache", Sys.BINDIR)
+    error("the wrong julia is being used: $(Sys.BINDIR)")
+end
+if VERSION >= v"1.7.0" # pkgdir was introduced here, and before then mtime wasn't a problem so just skip
+    using Pkg
+    src = pkgdir(Pkg, "src", "Pkg.jl")
+    # mtime is when it's compressed, ctime is when the file is extracted
+    if !<(mtime(src), ctime(src))
```

Comment:
Any reason you chose this over the more readable:
```suggestion
    if mtime(src) >= ctime(src)
```

Reply:
Because it matched the error message

Reply:
But I'll go for your suggestion for code readabilty